python/kwiver/sprokit/tests/sprokit/adapters/test-embedded_pipeline.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
-# ckwg +28
-# Copyright 2020 by Kitware, Inc.
+#ckwg +28
```

Comment:
Minor nit .. probably should keep that space in the copyright before "ckwg +28"

Also, here and above, did you mean to set the year back to 2011? At some point we should update the copyright headers to the new style anyway.